db/migrate/20210806105316_change_optionality_of_attributes.rb

```diff
@@ -14,5 +14,7 @@ def change
     change_column_null(:screenings, :date, true)
     change_column_null(:screenings, :movie_id, true)
     change_column_null(:screenings, :cinema_hall_id, true)
+
+    change_column_null(:reservation, :user_id, true)
```

Comment:
Remember that changing something in already processed migrations should be done rarely, only in specific situations. Here you don't have a production environment so you can do this, but remember to try not to do this in the future unless you will be sure that there is no other way to solve your issues. Here I'm pretty sure that you've had the ability to just create new migration and you were not forced to changing an existing one.

Reply:
Thanks, will remember to not do this :)